WebView/app/src/main/assets/main.js

```diff
@@ -25,7 +25,11 @@ function getData() {
 		var form = document.getElementById("location");
 		var currentLocation = form.options[form.selectedIndex].value;
 		document.getElementById("title").innerText = form.options[form.selectedIndex].text;
-        document.getElementById("description").innerText = data[currentLocation].description;
+        document.getElementById("currentTemp").innerText = data[currentLocation].currentTemp;
+        document.getElementById("shortDescription").innerText = data[currentLocation].description;
+        document.getElementById("longDescription").innerText = "Today in " + `${form.options[form.selectedIndex].text}`
```

Comment:
nit: use `currentLocation`

Reply:
Which piece needs to be switched? My guess is  the reference to `${form.options[form.selectedIndex].text}` in line 30, that is accessing a capitalized version of the location because current location is camel case. For example currentLocation is sanFrancisco and the other value is San Francisco.

Reply:
I see, that's fine then.